setup.py

```
@@ -38,7 +38,7 @@
             'opencv-contrib-python>=4.4.0.42',
             'opencv-python>=4.4.0.42',
             'grpcio',
```

can we remove this?

---

We need it to catch the grpc.Timeout exception. Alternatively we could just catch all exceptions or raise a custom exception thrown by engine.